tests/integration/dhcp_test.py

```diff
@@ -148,6 +163,43 @@ def test_dhcp_with_addresses(dhcp_env):
     assertlib.assert_state(desired_state)
 
 
+def test_dhcp_for_bond_with_ip_address_and_slave(dhcp_env,
+                                                 setup_remove_bond99):
+
+    desired_state = {
+        INTERFACES: [
+            {
+                'name': 'bond99',
+                'type': 'bond',
+                'state': 'up',
+                'ipv4': {
+                    'enabled': True,
+                    'dhcp': True,
+                    'address': [
+                        {'ip': IPV4_ADDRESS1, 'prefix-length': 24}
+                    ]
+                },
+
+                'link-aggregation': {
+                    'mode': 'balance-rr',
+                    'slave': [DHCP_CLI_NIC],
+                    'options':
+                        {'miimon': '140'}
+                }
+
+            },
+            {
+                'name': DHCP_CLI_NIC,
+                'type': 'ethernet',
+                'state': 'up'}
```

Comment:
The closing parenthesis needs to appear in the next line.

Reply:
done